includes/multi-currency/Settings.php

```diff
@@ -269,14 +269,17 @@ public function get_currency_setting( $currency ) {
 			$converted_rate = number_format( $available_currencies[ $currency->get_code() ]->get_rate(), 4 ) . ' ' . $currency->get_code();
 		}
 
+		$last_updated = ! is_null( $available_currencies[ $currency->get_code() ]->get_last_updated() )
+			? gmdate( 'H:i', $available_currencies[ $currency->get_code() ]->get_last_updated() ) . ' UTC'
```

Comment:
I'm wondering if we should just use the formatting, including date, from the WordPress settings here. Adding the date would be beneficial since a time could be any day, and using the format from WP would help the merchant/support to not have to do time zone conversions.

Reply:
Good call! I updated this in 63cf88d4b3f6ce416f8e0a4db998d31dc5158d30. Could you take another look and let me know? It should now display according to your WP install's time format/date format and in the timezone set in the WP settings.